src/handlers/int-test-support/helpers/s3Helper.ts

```diff
@@ -32,20 +33,44 @@ interface DataAndTarget {
   target: S3Object;
 }
 
+interface deleteS3ObjectByKeys {
+  bucket: string;
+  keysToDelete: ObjectIdentifier[];
```

Comment:
I think you can save the `toDelete` as the function name makes that clear.

Reply:
Also it's not the keys alone, also a potential version inside `ObjectIdentifier`. Why did we change that to `ObjectIdentifier`?

Reply:
I used objectIdentifier instead of plain keys it provides both key and version Id of the object where as keys would only delete latest version of the object

Reply:
change the type for keys to string[]